doc/foreign-code.rst

```diff
@@ -195,10 +195,10 @@ demonstrates how to access the bound C library functions and values:
 From here, one only needs to run ``dune build ./foo.exe`` to generate the
 stubs and build and link the example ``foo.exe`` program.
 
-Complete information about the ctypes combinators used above is available at
-the ctypes_ project.
+Complete information about the ``ctypes`` combinators used above is available at
+the ``ctypes_ project``.
```

Comment:
I believe this destroys the link to the Ctypes project.

Reply:
It apparently does. I'm looking to see if there is a way to put it both in monospace and keep the link...